rules/linux/lnx_process_discovery.yml

```diff
@@ -0,0 +1,22 @@
+title: Process Discovery
+id: 4e2f5868-08d4-413d-899f-dc2f1508627b
+status: stable
+description: Detects process discovery commands
+author: Ömer Günal, oscd.community
+date: 2020/10/06
+references:
+    - https://attack.mitre.org/techniques/T1057/
```

Comment:
could you please use [this](https://github.com/Neo23x0/sigma/pull/1044#discussion_r502045513) recommendation here as well?